### PR TITLE
docs: Add crash report architecture and implementation

### PR DESCRIPTION
**Pull request by marknguyen1302**
## Describe Your Changes

- Add crash report architecture
- Add crash report implementation
- Add crash report schema

## Fixes Issues

- Closes #
- Closes #

## Self Checklist

- [ ] Added relevant comments, esp in complex areas
- [ ] Updated docs (for bug fixes / features)
- [ ] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

**Comment by irfanpena** [comment]
@marknguyen1302 Hi Mark, thank you for this. I can help revamp and reformat this doc into a proper one that you can review later on. Also may ask you questions for content clarity

File: docs/telemetry.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    - Disable by set environment variables
+        - `CORTEX_CRASH_REPORT=0`
+    - Export the Telemetry to OpenTelemetry Collector:
+        - `OTEL_EXPORTER_OTLP_ENDPOINT=<endpoint>`
+
+- Commands:
+    - Show all crash reports:
+        - `cortex telemetry --type crash`
+
+## Type of Telemetry
```
Comment (0xSage) [inline docs/telemetry.md:34]: Types

Comment (0xSage) [inline docs/telemetry.md:34]: I'd move this section first, and rename it to Telemetry Collected.

**Review by 0xSage — APPROVED** [review]
Maybe @irfanpena can help to structure these docs to be a bit more explanatory. Otherwise, lgtm

**Review by 0xSage — APPROVED** [review]
great job

**Pull request merged**